docs/Syntax-Reference/draw.md

```diff
@@ -478,6 +478,8 @@ layers:
                     order: 2   # this layer's order is now 2
 ```
 
+`order` only affects the drawing order of features that are at the same elevation. If you have a rooftop drawn at 5m elevation with `order: 1` overlapping a road drawn at 0m elevation with `order: 2`, the rooftop will still be drawn on top. If you have two overlapping roads drawn at 0m elevation, the one with the higher `order` will be drawn on top.
```

Comment:
By `elevation` here are we talking [`z`](https://tangrams.readthedocs.io/en/master/Syntax-Reference/draw/#z)?

Reply:
Yeah, it's referring to the `z` value that the geometry ends up with, as distinct from the `z` draw parameter - since that final value can result from any/all of the `z` parameter, the `extrude` parameter, or vertex shader logic. Just saying `z` felt potentially ambiguous to me, but there could be a better term.